label_studio/fsm/serializers.py

```diff
@@ -23,6 +23,20 @@ class StateModelSerializer(serializers.Serializer):
     created_at = serializers.DateTimeField(read_only=True)
     context_data = serializers.JSONField(read_only=True)
 
+    def to_representation(self, instance):
+        """
+        Override to exclude triggered_by field for annotators and reviewers.
+        """
+        data = super().to_representation(instance)
+
+        request = self.context.get('request')
+        if request and hasattr(request, 'user'):
+            user = request.user
+            if getattr(user, 'is_annotator', False):
+                data.pop('triggered_by', None)
```

Comment:
This says it doesn't include the `triggered_by` for annotators or reviewers, but checks `is_annotator`  only.

Reply:
Yeah I re-enabled it for reviewers since I see they have access to the annotator column which has info about that user type. I didnt update the comment. I can either put the check for reviewer back or update the comment

Reply:
Yeah I re-enabled it for reviewers since I see they have access to the annotator column which has info about that user type. I didnt update the comment. I can either put the check for reviewer back or update the comment